maps_dashboards/public/components/add_layer_panel/add_layer_panel.tsx

```diff
@@ -40,23 +40,45 @@ export const AddLayerPanel = ({ setIsLayerConfigVisible, setSelectedLayerConfig
         opacity: 1,
         visibility: LAYER_VISIBILITY.VISIBLE,
       });
+    } else if (layerItem === DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER) {
+      setSelectedLayerConfig({
+        iconType: 'document',
+        name: DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER,
+        type: DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENT_LAYER,
+        id: uuidv4(),
+        zoomRange: [0, 22],
```

Comment:
Shall we move zoom range values to constant

Reply:
Yes, will update